src/main/resources/track.css

```diff
@@ -0,0 +1,8 @@
+body

```

Comment:
add css, that i think you project needs, didn't want to do to much because you guys may choose to do something different